.github/workflows/ci.yml

```diff
@@ -152,7 +160,43 @@ jobs:
       - name: Archive profiler artifacts
         uses: actions/upload-artifact@v3
         with:
-          name: profiler-${{ matrix.artifact-name || matrix.os }}
+          name: profiler-${{ matrix.artifact-name }}${{ steps.version.output.suffix }}
           path: |
             target/release/profiler${{ matrix.os == 'windows-latest' && '.exe' || '' }}
           if-no-files-found: error
+
+  release:
+    name: Publish release
+    if: github.event_name == 'push' && github.ref_type == 'tag'
+    needs: build
```

Comment:
IMHO, it should also need `test` to prevent non-functional releases.

Reply:
Yes, but `build` already needs `test`, so I didn't specify it.
<img width="1353" alt="image" src="https://github.com/spacemeshos/post-rs/assets/1897530/5652d0ef-66a6-4031-9569-9f75aa0d9f87">
(https://github.com/spacemeshos/post-rs/actions/runs/4970048025)

Or do you mean it's better to specify it explicitly once again just to be sure?

Reply:
Ok, I missed it